test/smoke/smoke_test.go

```diff
@@ -89,14 +96,37 @@ var _ = Describe("Smoke test", func() {
 
 			// Verify RF SF is met
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
+
```

Comment:
Considering we are giving customers like HP ability to run these test cases on their local environment we should move LM test into its own suite as they are reliant on s3 access. Please create a new suite for LM Test.